src/app/modules/shared_models/contacts_model.nim

```diff
@@ -124,7 +125,16 @@ QtObject:
     self.items[ind] = item
 
     let index = self.createIndex(ind, 0, nil)
-    self.dataChanged(index, index, @[]) # all roles
```

Comment:
@jrainville using empty roles array doesn't update all roles for the item? That's weird since see [documentation](https://doc.qt.io/qt-5/qabstractitemmodel.html#dataChanged).

Reply:
I assume NimQml doesn't handle it correctly